src/actions/SlsAction.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace asasmoyo\yii2saml\actions;
+
+use yii\base\Exception;
+use yii\base\InvalidConfigException;
+use yii\web\Response;
+
+
+/**
+ * This class handles saml logout request and response.
+ */
+class SlsAction extends BaseAction
+{
+    /**
+     * After succesfull logout process user will be redirected to this url.
+     * @var string
+     */
+    public $successUrl;
+
+    /**
+     * It handles sls logout request/response from Identity Provider. It will check whether is valid or not. If it isn't, an Exception will be thrown. If is valid, the successCallback will be called. You can use the callback to create user from attributes sent by Identity Provider or do something else. After that, user will be redirected to successUrl.     * @return $this|mixed
```

Comment:
please check this comment, i don't think we need callback here. and if it is, i don't see any successCallback definition here.